meson.build

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
so.. would we get rid of makefile? we don't have the budget of AOSP to support 15 different build systems within the same tree

@lumag?

Reply:
I'd say, there should be one building system for all the targets